test/e2e/certificates_test.go

```diff
@@ -171,10 +185,109 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate on ibm cloud CIS", func() {
+			cisCRN, isCisCRN := os.LookupEnv(cisCRNEnvironmentVar)
+			if targetPlatform, ok := os.LookupEnv(targetPlatformEnvironmentVar); ok && targetPlatform == "ibmcloud" {
+				if !isCisCRN || cisCRN == "" {
```

Comment:
If I'm not mistaken, you can also get the value of `cisCRN` from `infra.Status.PlatformStatus.IBMCloudPlatformStatus.CISInstanceCRN`. Will that value work the needs of the webhook that is being introduced?

Reply:
This is available in the Platform object from the IPI IBMCloud VPC. It's not available to the UPI PowerVS install.

Reply:
Same comments... https://github.com/openshift/cert-manager-operator/pull/170/files#r1523125878